impl/sql/mutationstorage/queue.go

```diff
@@ -104,19 +114,19 @@ func (m *Mutations) ListLogs(ctx context.Context, directoryID string, writable b
 	var logIDs []int64
 	rows, err := m.db.QueryContext(ctx, query, directoryID)
 	if err != nil {
-		return nil, err
+		return nil, ktsql.Errorf(err, "Query writable logs")
```

Comment:
nit: Not necessarily writeable. Same below.
```suggestion
		return nil, ktsql.Errorf(err, "Query logs")
```